docs/src/modules/components/ApiPage/sections/ClassesSection.tsx

```diff
@@ -64,10 +67,12 @@ export default function ClassesSection(props: ClassesSectionProps) {
     level: Level = 'h2',
     displayClassKeys,
     styleOverridesLink,
+    defaultLayout,
+    layoutStorageKey = API_LAYOUT_STORAGE_KEYS.classes,
```

Comment:
Maybe we should have the default values for `layoutStorageKey` defined in `ApiPage`. I imagine if we override them, we want people to override them for the whole page.